Unity3D/Assets/RosSharp/Scripts/ImageManager.cs

```diff
@@ -1,4 +1,24 @@
-﻿using RosSharp.RosBridgeClient;
```

Comment:
Is there ar reason why you added empty lines 4, 8 an 10 in the header? 
Please keep the headers as specified for the apache 2 license.